docs/monthly-meeting/2023-09.md

```diff
@@ -0,0 +1,118 @@
+  - Python docs don't fit into free open source tier on Weblate – the initial proposition is 1500€/year (50% discount)
+  - There is an option to self-host a Weblate instance
+  - (Both platforms support using machine learning translation tools)
+
+- [Ege] New bootstrapper for starting new Python docs translations
+  - Issues with the [current tool](https://github.com/JulienPalard/python-docs-cookiecutter):
+    - Unmaintained / inactive
+    - Uses third-party modules
+  - Add a link from the [devguide](https://devguide.python.org/documentation/translating/#starting-a-new-translation) to this [repository](https://github.com/egeakman/python-docs-bootstrapper)
+  - [PEP 545](https://peps.python.org/pep-0545/#repository-for-po-files) mentions the per-language repositories for `.po` files
+
+- [CAM] Quick update on reStructured Data
+  - Summary
+    - Mechanism to extract, process and output structured, machine-readible data from the Python docs
+    - Initial focus on deprecations/removals per user requests, then additions/changes and finally other annotations (supported platforms, stable ABI, audit events, etc.)
+    - Output a JSON per version and language for tool and API consumption
+    - Also output tables of all/specific deprecations/removals (or other changes) in the What's New and on a new dedicated Deprecations/Removals page.
+  - Progress
+    - Further considerable work on design document, schema, etc.
+    - Developed initial working implementation and prototyped/tested most planned features
+    - Discussed current approach at some length with `@AA-Turner` and currently making some significant changes based on his feedback
+    - Next steps: Re-scoping to focus on the basic features needed
+  - Learn more
+    - [HackMD detailed design document and schema](https://hackmd.io/jF8awEd3TuKh1Ia5Hqa_8g?view#TransformsProcessors)
+    - [Initial working prototype implementation](https://github.com/python/cpython/compare/main...CAM-Gerlach:cpython:add-structured-deprecation-support)
+
+- [Hugo] Python 3.12.0 is out on 2023-10-02! Any finishing touches needed for [What’s New In Python 3.12](https://docs.python.org/3.12/whatsnew/3.12.html)?
+  - [CAM] Was planning to finish this at the core dev sprints, but didn't realize the final release date was so much earlier than last year
+  - [CAM] Wanted to be able to use the new reStructured Data directive to list deprecations, removals, additions and changes in the appropriate places following the existing format, but too late for that now
+  - [CAM] May be able to use it to help guide the process, but timeline is going to be tight
+
+- [Daniele] How much appetite/consensus is there for an iterative workflow? [Discussion in Discord](https://discord.com/channels/935215565872693329/935215566334079058/1138743377962225794)
+  - Daniele can adapt the Discord messages into a Discourse thread and post that for further feedback and consensus
```

Comment:
I think @encukou suggested he may have the time to do this, I can't remember if it was decided though

Reply:
It's in my TODO pile, but I wouldn't mind if Daniele gor to it first :)

Reply:
Added you both